book/website/communication/outreach/outreach-effectively.md

```diff
@@ -0,0 +1,104 @@
+(cm-outreach-effectively)=
+# Communicating Effectively
+
+We should communicate and coordinate effectively with different audiences in an accessible and diverse way: journalists, the general public, industry partners, data scientists from different teams {cite}`NationalAcademiesofSciences2016comms`.
+
+## Researcher versus Journalist
```

Comment:
Researcher vs journalist feels like a bit of a false dichotomy between researchers and journalists - suggest the distinction between science writing and science journalism as an alternative?

```suggestion
## Science Writing versus Science Journalism
```